lib/takeaway.rb

```diff
@@ -0,0 +1,38 @@
+class Takeaway
+  def initialize(menu)
+    @select = {}
+    @menu = menu
+    @estimate = 0
+  end
+
+attr_reader :menu, :estimate
+
+  def show_menu
+    menu.show
+  end
+
+  def select(order, estimate)
+    @estimate = estimate
+    @select = order
+
+  end
+
+  def correct_bill?
+    raise "incorrect bill amount" unless calculate_bill == estimate
+    confirm_order
+  end 
+
+  def confirm_order
+    "Thank you! Your order was placed and will be delivered before #{delivery_time}"
```

Comment:
Metrics/LineLength: Line is too long. [84/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)